bigquery_etl/cli/query.py

```diff
@@ -478,13 +482,11 @@ def _backfill_query(
 
     backfill_date = backfill_date.strftime("%Y-%m-%d")
     if backfill_date not in exclude:
-        if no_partition:
-            destination_table = table
-        else:
-            destination_table = f"{table}${partition}"
+        if not no_partition:
+            destination_table = f"{destination_table}${partition}"
```

Comment:
I'd add a comment just because the double negative is a little weird
```suggestion
        # If table is partitioned, add partition decorator: 
        if not no_partition:
            destination_table = f"{destination_table}${partition}"
```